### DISCUSSION TITLE
Pip release of  Version 1

### DISCUSSION THREAD

**Discussion by NilsB98** [open-edge-platform/anomalib#1676] [General]
Hi together, 
I just wanted to ask when the new release of V1 will be released on pip, since it seems that the `pip install anomalib` command installs the 0.7 version.
Or did I misunderstand the v1 and this isn't a new release?

I hope this is the correct place to ask this question :)

Best regards.

**Comment by blaz-r** [comment]
Hi @NilsB98. Yes, the v1 is a new release, but currently it is only available as direct install from github. I believe it will be soon available from pip as well.

**Reply by NilsB98** [reply]
Hi @blaz-r, okay thank you for the information and the quick response!
